.gitignore

```diff
@@ -3,6 +3,7 @@
 # dependencies
 /node_modules
 /apps/web/node_modules
+/apps/web/.cache
```

Comment:
thoughts on ignoring this? it was just generating an `llm-cache.json` empty object file, can also just add it in